core/src/main/scala/org/apache/spark/sql/delta/commands/WriteIntoDelta.scala

```diff
@@ -52,6 +52,10 @@ import org.apache.spark.sql.types.StructType
  * In combination with `Overwrite`, a `replaceWhere` option can be used to transactionally
  * replace data that matches a predicate.
  *
+ * In combination with `Overwrite` dynamic partition overwrite mode (option `partitionOverwriteMode`
```

Comment:
nit: mention this can also be set in the spark conf

Reply:
fixed